pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -878,61 +884,64 @@ private CompletableFuture<Subscription> getDurableSubscription(String subscripti
 
         Map<String, Long> properties = PersistentSubscription.getBaseCursorProperties(replicated);
 
-        ledger.asyncOpenCursor(Codec.encode(subscriptionName), initialPosition, properties, new OpenCursorCallback() {
-            @Override
-            public void openCursorComplete(ManagedCursor cursor, Object ctx) {
-                if (log.isDebugEnabled()) {
-                    log.debug("[{}][{}] Opened cursor", topic, subscriptionName);
-                }
+        ledger.asyncOpenCursor(Codec.encode(subscriptionName), initialPosition, properties, startMessageIdInclusive,
```

Comment:
Seems like only the code format has changed?

Reply:
Updated.